run_tests.py

```diff
@@ -69,6 +72,9 @@
             else:
                 shutil.copy(file_or_dir_src, dest)
 
+    print("\n\n------ UPGRADING configurations ------")
+    perform_upgrade()
```

Comment:
I get an error when this is run of `No such file or directory: "src\\schemas\\new_banner.xsd"`